src/libraries/System.Reflection.Emit/tests/TypeBuilder/TypeBuilderSetCustomAttribute.cs

```diff
@@ -52,6 +52,26 @@ public void SetCustomAttribute()
             Assert.Equal("hello", ((TypeBuilderStringAttribute)attributes[0]).Creator);
         }
 
+        [Fact]
```

Comment:
This test should be under https://github.com/dotnet/runtime/tree/main/src/libraries/System.Runtime/tests/System.Reflection.Tests . It is not testing anything specific to Reflection.Emit.